monix-eval/shared/src/main/scala/monix/eval/Coeval.scala

```diff
@@ -74,35 +74,35 @@ import scala.util.{Failure, Success, Try}
   * instances out of strict values:
   *
   * {{{
-  *   val fa = Coeval.now(1)
-  *   fa.value //=> 1
+  *   val faNow = Coeval.now(1)
+  *   faNow.value //=> 1
   *
-  *   val fe = Coeval.raiseError(new DummyException("dummy"))
-  *   fe.value //=> throws DummyException
+  *   val feThrowNow = Coeval.raiseError(new RuntimeException("dummy"))
+  *   feThrowNow.failed //=> has RuntimeException
   * }}}
   *
   * The "always" strategy is equivalent with a plain function:
   *
   * {{{
   *   // For didactic purposes, don't use shared vars at home :-)
   *   var i = 0
-  *   val fa = Coeval.eval { i += 1; i }
+  *   val faEval = Coeval.eval { i += 1; i }
   *
-  *   fa.value //=> 1
-  *   fa.value //=> 2
-  *   fa.value //=> 3
+  *   faEval.value //=> 1
+  *   faEval.value //=> 2
+  *   faEval.value //=> 3
   * }}}
   *
   * The "once" strategy is equivalent with Scala's `lazy val`
   * (along with thread-safe idempotency guarantees):
   *
   * {{{
-  *   var i = 0
-  *   val fa = Coeval.evalOnce { i += 1; i }
+  *   var j = 0
+  *   val faEvalOnce = Coeval.evalOnce { j += 1; j }
```

Comment:
Same question: why rename? `fa` is a typical name for "some effect with some value". I'd either keep it as is, or name it `coevalOnce` or sth like that.

Reply:
`coevalOnce` will do as long as it's not repeated in same Scaladoc